src/stravalib/model.py

```diff
@@ -351,33 +351,8 @@ class SummaryClub(MetaClub, strava_model.SummaryClub):
     dimensions: Optional[list[str]] = None
```

Comment:
Related to your other comment, these three attrs that i've added are odd and likely not sure useful to users. I have removed them but pls say the word if you think we should add them given i added them and it's just more for us to maintain.

<img width="486" alt="Screenshot 2024-06-12 at 12 22 25 AM" src="https://github.com/stravalib/stravalib/assets/7649194/7aaea8bd-7045-4faf-9f5d-43d044a214a1">